angular/projects/admin/src/app/services/document-type.service.ts

```diff
@@ -31,11 +31,11 @@ export class DocumentTypeService {
       .valueChanges();
   }
 
-  async create(id: string) {
-    const docRef = this.siteCollection.collection('document-types').doc(id);
+  async create(title: string, slug: string) {
+    const docRef = this.siteCollection.collection('document-types').doc(slug);
     return docRef.set({
-      id: id,
-      title: null,
+      id: slug,
+      title: title,
       description: '',
       slug: null,
```

Comment:
The slug should be saved to `slug` field also.

Reply:
Resolve on this commit https://github.com/oddbit/tanam/pull/59/commits/8fca9d95aa49ab745896b6bf3d23e3cd48589233